zallet/src/components/sync.rs

```diff
@@ -574,39 +574,44 @@ async fn data_requests(
                             .set_transaction_status(txid, TransactionStatus::TxidNotRecognized)?;
                     }
                 }
-                TransactionDataRequest::TransactionsInvolvingAddress {
-                    address,
-                    block_range_start,
-                    block_range_end,
-                    request_at,
-                    tx_status_filter,
-                    output_status_filter,
-                } => {
-                    let address = address.encode(params);
+                TransactionDataRequest::TransactionsInvolvingAddress(req) => {
+                    info!(
+                        "Checking for new UTXOs at address {}",
+                        req.address().encode(params)
+                    );
+
```

Comment:
Do we need this in addition to the log line below? I made that one `debug!` in 1606318fecf8d058961f6e90731fca471a6f04fd to reduce noise, and I think this `info!` line is still going to be somewhat noisy (even if only occurring on each new block), or at least not much less noisy than the `debug!` line (in which case we could just make that one `info!` again instead of logging twice in debug mode).

Reply:
Removed.